src/components/challengeWidget/challengeWidget.css

```diff
@@ -32,13 +33,13 @@
 }
 
 .challengeWidget .markers {
+  font-family: inherit;
```

Comment:
Given that `.markers` is a `<div>` it's already inheriting `font-family`, I think there's no need for this.